lib/Model.js

```diff
@@ -141,6 +141,16 @@ Model.prototype._collectRatio = 0.75;
  */
 Model.prototype.get = require("./response/get");
 
+Model.prototype._getOptimizedBoundPath = function _getOptimizedBoundPath() {
```

Comment:
Don't we have a copy already?

Reply:
Copy of what?

Reply:
https://github.com/Netflix/falcor/blob/master/lib/support/array-clone.js

Reply:
I would imagine this function being a touch more like 
```
const clone = require('./path/to/array-clone');

Model.prototype._getOptimizedBoundPath = function _getOptimizedBoundPath() {
    return clone(this._path);
}
```

Reply:
ah cool